docs/usage/applications/insecure-configuration/_index.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Auditing Kubernetes for Secure Setup
+description: "A few insecure configurations in Kubernetes"
```

Comment:
Probably a better place for this document is https://github.com/gardener/community.